src/darwin/Framework/CHIP/MTRError_Internal.h

```diff
@@ -26,6 +26,10 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+#ifndef YES_NO
+#define YES_NO(x) ((x) ? @"YES" : @"NO")
```

Comment:
This seems added but never used.

readability-wise, is this typical style? reading code of `YES_NO(foo)` is not immediately obvious for being a NSSTRING ... maybe `YES_NO_STR` or something like that would be more obvious?